src/Microsoft.AspNet.OData.Shared/Builder/StructuralTypeConfiguration.cs

```diff
@@ -143,6 +144,22 @@ public PropertyInfo DynamicPropertyDictionary
             get { return _dynamicPropertyDictionary; }
         }
 
+        /// <summary>
+        /// Gets a value indicating whether this type has instance annotations or not.
+        /// </summary>
+        public bool SupportsInstanceAnnotations
```

Comment:
>SupportsInstanceAnnotations [](start = 20, length = 27)

IncludeInstanceAnnotations

Reply:
I'm fine with "SupportsInstanceAnnotations" -- it doesn't mean that there are instance annotations, nor that instance annotations should be written, just that the type is capable of exposing instance annotations.

---
In reply to: [488997684](https://github.com/OData/WebApi/pull/2219#discussion_r488997684) [](ancestors = 488997684)